.github/workflows/unit_test.yml

```diff
@@ -56,3 +56,30 @@ jobs:
           GOBIN: /home/runner/go/bin
     - name: Run 
       run: make test-verbose
+
+  integration_test:
+    runs-on: ubuntu-latest
+    needs: unit_test_without_bcc
+    steps:
+      - uses: actions/checkout@main
+      - uses: actions/setup-go@main
+        with:
+          go-version: 1.18
+      - name: install kind
+        run: go install sigs.k8s.io/kind@v0.12.0
+      - name: check version
+        run: kind version
+      - name: install kubectl
+        run: curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+      - name: check version
+        run: kubectl version || true
+      - name: build image
+        run: export CTR_CMD=docker && make
+      - name: infra up
+        run: ./scripts/kind.sh
+      - name: upload image to local registry
+        run: kind load docker-image quay.io/sustainable_computing_io/kepler:latest
```

Comment:
> will this command load the image into docker registry or kubernetes/openshift registry?
this command will load image into local docker registry for KIND(k8s in docker cluster)

Reply:
> do you need to load it locally? it will be downloaded when deploying kepler...
yes, we need to load it into local cluster.
as
for PR level, even if we don't have any integration test case for now, but
the logic for this PR at integration part is:
- download tools as KIND(k8s in docker) and kubectl.
- build kepler image with specific pr code.
- start KIND cluster locally with a local image registry.
- upload kepler image build by specific pr code to local image registry for kind cluster.
- deploy a kepler at local kind cluster with image stored at local image registry.
- check via kubectl command for ...